environment.go

```diff
@@ -238,12 +238,12 @@ func NewEnvironment(ctx context.Context, confPath, versionID string, db *DBSetti
 	catcher.Add(e.initJasper(ctx, tracer))
 	catcher.Add(e.initDepot(ctx, tracer))
 	catcher.Add(e.initThirdPartySenders(ctx, tracer))
+	catcher.Add(e.initClientConfig(ctx, versionID, tracer))
```

Comment:
since we only add this to a catcher and don't fail, we should still probably do a nil check in initClientConfig to avoid a panic if this fails (unless we don't do that kind of thinking in any of the other functions in which case disregard 🤷‍♀️)

Reply:
I believe the ticket discourages that behavior because this is an edge case and it wouldn't be efficient to add a nil check for every time we use the client for it.